GliaWidgets/SecureConversations/Confirmation/SecureConversations.ConfirmationStyle.swift

```diff
@@ -0,0 +1,103 @@
+import Foundation
+import UIKit
+
+extension SecureConversations {
+    public struct ConfirmationStyle: Equatable {
```

Comment:
let's add short doc for struct also.

Reply:
Fixed 3368b8d40db9b10677075b20b9718ad45ea5e7cf